netjsonconfig/backends/vxlan/schema.py

```diff
@@ -0,0 +1,46 @@
+from copy import deepcopy
+
+from ..wireguard.schema import schema as base_schema
+
+base_vxlan_properties = {
+    "vxlan": {
+        "type": "array",
+        "title": "VXLAN",
+        "uniqueItems": True,
+        "additionalItems": True,
+        "propertyOrder": 13,
+        "items": {
+            "type": "object",
+            "title": "VXLAN tunnel",
+            "additionalProperties": True,
+            "properties": {
+                "name": {
+                    "title": "interface name",
+                    "description": "VXLAN interface name",
+                    "type": "string",
+                    "minLength": 2,
+                    "maxLength": 15,
+                    "pattern": "^[^\\s]*$",
+                    "propertyOrder": 1,
+                },
+                "vni": {
+                    "type": "integer",
+                    "title": "VNI",
+                    "description": (
+                        "VXLAN Network Identifier, if set to \"0\", each tunnel will have"
+                        " different VNI. If a non-zero VNI is specified, then it will be"
+                        " used for all VXLAN tunnels."
+                    ),
```

Comment:
As far as I know zero is a possible VNI, some networking equipment allow using this, others start from 1.
If we don't want to hit limitations in corner case we have to look for an alternative, we could use a switcher here (oneOf).